src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random randomInt = new Random();
```

Comment:
why static?
```suggestion
    private final Random random = new Random();
```

Reply:
Thanks for your question. To save some memory. As I understand, I don't need to create new Random for every instance of Lottery. Thats why I've used static final. Is it not good?